.github/ISSUE_TEMPLATE/gui_issue.yml

```
@@ -0,0 +1,18 @@
+name: Issue or feature request related to the GUI
+description: Any report, issue or feature request related to the GUI
+labels: [GUI]
+body:
+- type: checkboxes
```

YAMLlint (1.37.1)

[error] wrong indentation: expected at least 1

(indentation)